src/tmx.js

```diff
@@ -592,6 +602,22 @@ class TMX {
                                     tu.sourceLocale = variant.locale;
                                 }
                             }
+                            if(pre){
+                                const parsedPre = xmljs.xml2js(pre, {
+                                    trim: false,
```

Comment:
I take it that the content of the pre and post properties is xml that is xml-encoded, so that is why we have to call the xml2js again on these? Will these snippets of xml always be well-formed? I'm not sure what will happen if xml2js is given non-well-formed xml...